README.md

```diff
@@ -1,11 +1,48 @@
-# sinol-make
-CLI tool for creating sio2 task packages. \
-Currently in development and not yet ready to be used.
+# sinol-make ![](https://avatars.githubusercontent.com/u/2264918?s=200&v=4)
 
-## Installing from source
-`pip3 install .`
+`sinol-make` is a CLI tool for creating and verifying problem packages
+for [sio2](https://github.com/sio2project/oioioi)
+with features such as:
+- running the solutions in parallel,
+- keeping a git-friendly report of solutions' scores,
+- catching mistakes in the problem packages as early as possible,
+- and more.
+
+# Contents
+
+- [Why?](#why)
+- [Installation](#installation)
+- [Usage](#usage)
+- [Configuarion](#configuration)
+- [Reporting bugs and contributing code](#reporting-bugs-and-contributing-code)
+
+### Why?
+
+The purpose of the tool is to make it easier to create good problem packages
+for official competitions, which requires collaboration with other people
+and using a multitude of "good practices" recommendations.
+While there are several excellent CLI tools for creating tests and solutions,
+they lack some built-in mechanisms for verifying packages and finding mistakes
+before uploading the package to the judge system.
+
+### Installation
+
+It's possible to directly install [sinol-make](https://pypi.org/project/sinol-make/)
+through Python's package manager pip, which usually is installed alongside Python:
+
+`pip3 install sinol-make`
+
```

Comment:
```suggestion

`pip3 install sinol-make`

We support `macOS` through the usage of `time` instead of `oiejq`.
As using `oiejq` is highly recommended, we suggest using a Linux-based
operating system whenever possible.

```

Reply:
`oiejq` doesn't also work on WSL on Windows, so we should add it here

Reply:
I've updated the suggestion. If you think it's good, you can clik on the "commit"